design.md

```diff
@@ -1,13 +1,20 @@
 # Design
 
 ## Who is the target for this design, e.g., are you assuming any knowledge on the part of the language users?
+The target of this design is to mimic language used by regular people to direct someone across a space. I am not assuming much technical knowledge of the users - the script should look like someone reading a list of directions or rules to navigate someone who is blind(folded).
```

Comment:
Looking at the sample codes, I think that you do meet this! But it could be more natural language like, if there was more options? Like, more ways to phrase a certain thing.

Reply:
[with natural languages, you can phrase the same thing in multiple different ways - but the provided implementation is quite restrictive. It could be mimic regular language even better if it had more flexibility]